.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,210 @@
+   # Navigate to the newly cloned directory
+   cd ~/your-local-git-repo-spot/over_react
+   # Assign the repo you forked from to a remote called "upstream"
+   git remote add upstream git@github.com:Workiva/over_react
+   ```
+
+2. If you cloned a while ago, get the latest changes from upstream:
+
+   ```bash
+   git checkout master
+   git pull upstream master
+   ```
+
+3. Create a new topic branch that will contain your feature, change, or fix:
+
+   ```bash
+   git checkout -b <topic-branch-name>
+   ```
+
+4. Commit your changes in logical chunks. Please adhere to these [git commit message guidelines](#git-commit-message-standards) or your code is unlikely be merged into the master branch. Optionally, you can use Git's [interactive rebase](https://help.github.com/articles/interactive-rebase) feature to tidy up your commits before making them public.
+
+5. Write tests for your changes.  
+  1. There are no exceptions.  
+  2. If your PR doesn't have accompanying test coverage - your code 
+     will not get merged.
+
+6. Locally merge the upstream master branch into your topic branch:
+
+   ```bash
+   git pull upstream master
+   ```
+
+7. Push your topic branch up to your fork:
+
+   ```bash
+   git push origin <topic-branch-name>
+   ```
+
+8. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
+    with a clear title and description - following all the [issue guidelines](#using-the-issue-tracker) listed above.
+
+&nbsp;
+&nbsp;
+
+
+
+## Git Commit Message Standards
+
+Below you will find an example commit message that follows the guidelines we would like all over_react contributors to follow.
+
+```
+Capitalized, short (50 chars or less) summary
+
+More detailed explanatory text, if necessary.  Wrap it to about 72
+characters or so.  In some contexts, the first line is treated as the
+subject of an email and the rest of the text as the body.  The blank
+line separating the summary from the body is critical (unless you omit
+the body entirely); tools like rebase can get confused if you run the
+two together.
+
++ Bullet points are okay, too
+
++ Typically a hyphen, asterisk or plus-symbol is used for the bullet, 
+followed by a single space, with blank lines in between, but 
+conventions vary here
+```
+
+Write your commit message in the imperative: "Fix bug" and not "Fixed bug" or "Fixes bug."  This convention matches up with commit messages generated by commands like git merge and git revert.
+
+Further paragraphs come after blank lines.
+
+> [Read this](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more information on why standardized commit message format is important.
+
+&nbsp;
+&nbsp;
+
+
+
+## Developer Workflow
+
+The `over_react` developer workflow couldn't be any more simple!
+
+```bash
+pub serve
+```
+
+When you're ready to run the tests... in a separate terminal:
+
+```bash
+pub run test -p content-shell --pub-serve=8081 test/over_react_test.dart
+```
```

Comment:
There are the VM tests that we want to run as well, but wouldn't get run using these instructions. We should just recommend using `pub run dart_dev test`.